lib/card.rb

```diff
@@ -14,11 +14,9 @@ class Card < ActiveRecord::Base
 
   has_many :references_from, :class_name => :Reference, :foreign_key => :referee_id
   has_many :references_to,   :class_name => :Reference, :foreign_key => :referer_id
-  has_many :acts, :order => :id
-  has_many :actions, :order => :id, :conditions=>{:draft => [nil,false]}
-  has_many :drafts, :order=>:id, :conditions=>{:draft=>true}, :class_name=> :Action
-
-  cache_attributes 'name', 'type_id' # review - still worth it in Rails 3?
+  has_many :acts, -> { order :id }
+  has_many :actions, -> { where( :draft=>[nil,false]).order :id }
+  has_many :drafts, { :class_name=> :Action }, -> { where( :draft=>true ).order :id }
```

Comment:
fyi, I will still probably start by getting it to work on 4.2 and then trying to connect the dots.  It would be my preference to try to keep this to one version for now.  Perhaps what we'll do is jump to requiring ~> 4 at Decko 1.0 but, as you suggest, continue supporting old 3.2.x versions in limited Decko 0.x releases?